src/styles/theme/themes/dark.ts

```diff
@@ -87,6 +88,8 @@ const darkTheme = {
     starDefaultBG: 'rgb(254, 228, 94)',
     mapAttributionText: colors.black,
     white: colors.white,
+    progressBarBackground: colors.productDark400,
+    progressBarFill: colors.green400,
```

Comment:
Not sure how we name our colors but are we able to just grab the "success" color here instead of a specific color value for green?

Reply:
We are, thank you for pointing that out. I've replaced this one with `colors.success`.